Podcast/FeedViewController.swift

```diff
@@ -100,18 +109,22 @@ class FeedViewController: ViewController {
             self.feedTableView.endRefreshing()
             self.feedTableView.stopLoadingAnimation()
             self.feedTableView.finishInfiniteScroll()
-            self.feedTableView.reloadData()
+            self.reloadTableView()
         }
         
         fetchFeedEndpointRequest.failure = { _ in
             self.feedTableView.stopLoadingAnimation()
             self.feedTableView.endRefreshing()
             self.feedTableView.finishInfiniteScroll()
-            self.feedTableView.reloadData()
+            self.reloadTableView()
         }
         
         System.endpointRequestQueue.addOperation(fetchFeedEndpointRequest)
     }
+
+    func reloadTableView() {
+        feedTableView.reloadSections(IndexSet(integer: 0), with: .fade)
```

Comment:
not sure if the fade animation should be applied every time the view appears (such as every time you tap on the tab bar for feed), it's kind of distracting

Reply:
See above